src/_cffi_src/openssl/crypto.py

```diff
@@ -143,6 +143,7 @@
 }
 
 void Cryptography_free_wrapper(void *ptr, const char *path, int line) {
-    return free(ptr);
+    free(ptr);
+    return;
```

Comment:
You can just drop the `return` line, `void` functions don't need to return.

Reply:
Thanks, did just that.